.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "eslint.workingDirectories": [
+        "frontend"
+    ]
+}
```

Comment:
```suggestion
    "python.testing.unittestEnabled": false,
    "python.testing.pytestEnabled": true,
    "python.testing.cwd": "${workspaceFolder}/backend",
    "files.exclude": {
        "**/__pycache__": true,
        "**/.hypothesis": true,
        "**/.pytest_cache": true
    }
}
```

Reply:
I also second this addition (will just have to make sure you add the missing `,` to the previous item)